operations/research-and-development/product/release-process/release-tips.md

```diff
@@ -46,9 +46,4 @@
 **Release Marketing**
 
  - Blog post:
-    - Follow [guidelines](https://docs.mattermost.com/process/marketing-guidelines.html#guidelines-for-release-announcements).
-    - Audience: end-users, admin/tech.
```

Comment:
Did you intentionally remove the audience and other lines?

Reply:
Yes.

Reply:
Makes sense, we have broader guidelines for it now